cloudbees-cd/es-to-os-migration/update-es-role-permissions.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+### Run this script using the system user assigned to run the CloudBees Analytics server
+### If you are not the user assigned to run the CloudBees Analytics, you can use `sudo su` to impersonate that user.
+### For example: if system user is analytics ###
+#
+# sudo su analytics
+#
+### end of example ###
+
+### Specify custom variables as environmental variables if required for your installation
+# ES_DIR_CONF - conf dir which contains installed config yml files
+# ES_DIR_BINS - dir with ElasticSearch binaries
+# ES_JAVA_HOME - dir with java
+# ES_PORT -  port to communicate ES, specify new port if it was not configured by default as 9300
+# CONFIG_DIR_TMP - temporary folder with write permissions
+
+### example ###
+#
+# ES_PORT=9300 ES_DIR_CONF=/opt/cb/sda/conf/reporting/elasticsearch ./update-es-role-permissions.sh
+#
+### end of example ###
+
+function show_help() {
+    echo "update_es_role_permissions.sh [-h]"
+    echo  "-h show help"
+
+    echo "****************************************************************"
+    echo  "Run this script using the system user assigned to run the CloudBees Analytics server"
+    echo  "If you are not the user assigned to run the CloudBees Analytics, you can use \`sudo su\` to impersonate that user."
+    echo "### For example: if system user is analytics ###"
+    echo ""
+    echo "sudo su analytics"
+    echo ""
+    echo "### end of example ###"
+    echo ""
+    echo "Specify custom variables as environmental variables if required for your installation"
+    echo "ES_DIR_CONF - conf dir which contains installed config yml files"
+    echo  "ES_DIR_BINS - dir with ElasticSearch binaries"
+    echo "ES_JAVA_HOME - dir with java"
+    echo "ES_PORT -  port to communicate ES, specify new port if it was not configured by default as 9300"
+    echo "CONFIG_DIR_TMP - temporary folder with write permissions"
+    echo ""
+    echo "### example ###"
+    echo ""
+    echo  "ES_PORT=9300 ES_DIR_CONF=/opt/cb/sda/conf/reporting/elasticsearch ./update-es-role-permissions.sh"
```

Comment:
The example either here or if there is another doc which will refer to this script should provide a more commonly used example. IMO, that will be where DOIS in installed in a non-default directory. 

If DOIS is installed in `/opt/cloudbees/dois`, then run:
`ES_DIR_CONF=/opt/cloudbees/dois/conf/reporting/elasticsearch ES_DIR_BINS=/opt/cloudbees/dois/reporting/elasticsearch/bin ES_JAVA_HOME=/opt/cloudbees/dois/reporting/jre  ./update-es-role-permissions.sh"`